lend/src/pages/LendPage.tsx

```diff
@@ -282,7 +282,7 @@ const Header = ({ setWalletModal }: HeaderProps) => {
       <HeaderRow>
         <MarqueeCol md={12} lg={6}>
           {/* <StatsMarquee /> */}
```

Comment:
Is the `StatsMarquee` still going to be used here? Otherwise we can remove it

Reply:
we will use these once we get higher values